packages/calypso-products/package.json

```diff
@@ -36,10 +36,12 @@
 	"homepage": "https://github.com/Automattic/wp-calypso/tree/HEAD/packages/calypso-products#readme",
 	"dependencies": {
 		"@automattic/calypso-config": "workspace:^",
+		"@automattic/shopping-cart": "workspace:^",
 		"i18n-calypso": "workspace:^"
 	},
 	"devDependencies": {
 		"@automattic/calypso-typescript-config": "workspace:^",
+		"react-dom": "^18.2.0",
```

Comment:
Should this maybe be a `peerDependency`? (I may be totally wrong)

Reply:
Good observation -- mulling this over 🤔

Reply:
I'm actually unsure here. My understanding is that peer dependencies specify compatible versions of a dependency for a package.

The `calypso-products` package itself doesn't need `react-dom` though, so I'm not sure it makes sense to "define" compatible versions. Otherwise, we'd need to add peer dependencies for any dependent packages with peer dependencies used by `calypso-products`

With that being I'm also not strongly certain about this either

Reply:
I don't honestly know enough about deps/peer deps to say. I was hoping you would know 😅 . In any case it's easily changed later if we find a better route.